lib/public/views/Runs/ActiveColumns/runDetectorsAsyncQcActiveColumns.js

```diff
@@ -12,6 +12,62 @@
  */
 import { h } from '/js/src/index.js';
 import { frontLink } from '../../../components/common/navigation/frontLink.js';
+import { formatFloat } from '../../../utilities/formatting/formatFloat.js';
+import { isContrastBlack } from '../../../components/common/badge.js';
+import { ChartColors } from '../../Statistics/chartColors.js';
+
+/**
+ * Render QC summary for given run and detector
+ * @param {number} runNumber run number
+ * @param {number} dplDetectorId id of the detector
+ * @param {QcSummary} qcSummary QC summary
+ * @return {Component} component
+ */
+const runDetectorAsyncQualityDisplay = (runNumber, dplDetectorId, qcSummary) => {
+    if (!qcSummary?.[runNumber]?.[dplDetectorId]) {
+        return h('.btn.white.bg-primary', 'QC');
```

Comment:
It would be great to here have a button that redirects the user straight to the creation of a flag for that (datapass,run,detector) and a `plusIcon`

Reply:
done